foodwebsite/style.scss

```diff
@@ -0,0 +1,361 @@
+
+$font-stack: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+$pimary-color:#FFF;
+
+.hb-header{
+    display: flex;
+    flex-flow: column wrap;
+    .header-wrapper-top{
+        height:55vh;
+        background-image: url('img/ChatGPT\ Image\ Jul\ 16\,\ 2025\,\ 09_54_09\ AM.png');
+        background-size: cover;
+        background-position: center center;
+        background-repeat: no-repeat;
+        .header-image-part{
+            display: flex;
+            flex-flow: column wrap;
+            .header-top{
+                display: flex;
+                flex-flow: row wrap;
+                align-items: center;
+                justify-content: space-between;
+                .logo{
+                    display: flex;
+                    flex-direction: row;
+                    align-items: center;
+                    margin-top: 40px;
+                    margin-left: 200px;
+                    display: flex;
+                    font-size: 20px;
+                    font-family:'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+                    letter-spacing: 1px;
+                    text-transform: uppercase;
+                    cursor: pointer;
+                    color: #FFF;
+                    font-weight:bold;
+                    .logo-text{
+                        margin: 5px;
+                        span{
+                            display: block;
+                            margin: 3px;
+                        }
+                    }
+                    .logo-icon{
+                        width: 40px;
+                        height: 40px;
+                        border-radius: 50%;
+                        object-fit: cover;
+                        top: -10px;
+                        left: -50px;
+                    }
+                }
+                .navigation{
+                    padding-right: 200px;
+                    padding-top: 10px ;
+                    .table-content{
+                        display: flex;
+                        flex-flow: row wrap;
+                        align-items: center;
+                        list-style-type: none;
+                        
+                        p{
+                            
```

Comment:
The indentations seem off. Can you please format the file? 
Also, I wouldn't nest all the elements, unless necessary 🤔